libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1931,6 +1931,10 @@ void GCS_MAVLINK::handle_common_mission_message(mavlink_message_t *msg)
         break;
     }
 
+    case MAVLINK_MSG_ID_AUTOPILOT_VERSION_REQUEST:
```

Comment:
This is in the wrong place right? It's inside `handle_common_mission_message`.

Reply:
Yes.  Fixed.